### ISSUE TITLE
Is it possible to trim an EditedMediaItemSequence to a specified number of seconds?

### ISSUE THREAD

**Issue by tomo-TW** [androidx/media#1419]
Hi,
Transformer has a feature to concatenate videos using `EditedMediaItemSequence`.
I have a question.
I can trim videos using `ClippingConfiguration `with a single `MediaItem`, but is it possible to trim videos with `EditedMediaItemSequence`?
From what I can see, there doesn't seem to be an API that would allow for such use cases.

**Comment by kim-vde** [comment]
Hi,

There is no API to do that at the moment.

Do you know the duration of the `MediaItems` in the sequence? If so, you could compute which `MediaItem` to clip and the end time.

**Comment by tomo-TW** [comment]
Yes, The playback start and end times of each `MediaItems` are managed using database records.
It seems that the requirement can be achieved by specifying the trimming time and subtracting it.
Thanks I will give it a try

**Comment by kim-vde** [comment]
Great, will close the issue then!
